docs/conf.py

```diff
@@ -153,7 +153,7 @@
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
 #
-html_theme = "qiskit"  # use the theme in subdir 'theme'
+html_theme = "qiskit-ecosystem"  # use the theme in subdir 'theme'
 
```

Comment:
You should set `html_title = f"{project} {release}"`

Reply:
Also the comment `# use the theme in subdir 'theme'` is bad copy and paste from ages ago

Reply:
Done.